NAMESPACE

```diff
@@ -162,6 +165,7 @@ importFrom(mgcv,predict.gam)
 importFrom(mgcv,qq.gam)
 importFrom(mgcv,residuals.gam)
 importFrom(mgcv,s)
+importFrom(mgcv,slanczos)
 importFrom(mgcv,smooth.construct)
 importFrom(mgcv,smooth.construct.ps.smooth.spec)
 importFrom(mgcv,smooth.construct.t2.smooth.spec)
```

Comment:
This probably needs to add

`importFrom("stats", "cmdscale")`

see https://travis-ci.org/refunders/refund/jobs/138205499